source/definitions.h

```diff
@@ -50,7 +50,7 @@
 #endif
 // OS
 
-#define OTGZ_SUPPORT 1
+//#define OTGZ_SUPPORT 1
```

Comment:
any specific reason for disabling otgz support?

Reply:
I ran into problems with linker in visual studio 2019 (the error is "unrecognized external symbol"):
![obraz](https://user-images.githubusercontent.com/5205079/142212161-0d095b75-a784-4c29-9285-a35b88c09c9f.png)

I can reenable it for pull request purposes though

Reply:
nvm, was bad configuration in my visual studio. I pushed a commit reverting that.